lib/ansible/playbook/base.py

```diff
@@ -106,7 +106,8 @@ def _generic_g(prop_name, self):
         if hasattr(self, method):
             return getattr(self, method)()
 
-        value = self._attributes[prop_name]
+        value = self._attributes.get(prop_name)
```

Comment:
This feels like there's something very wrong.  Was the former code throwing a KeyError?  If so, then .get() is just masking the problem (returning None instead of throwing a KeyError).  I think this is a bad change but would have to look at the error to know what the correct change is.